src/main.js

```diff
@@ -1,6 +1,7 @@
 import Vue from 'vue';
 import dotenv from 'dotenv';
 import App from './App.vue';
+import * as Sentry from '@sentry/browser'
```

Comment:
Absolute imports should come before relative imports  import/imports-first<br>Missing semicolon                                     semi